src/Pages/3DPrintingConsole/requestForm.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import './3D-console.css'
+import {
+  Button,
+  ButtonGroup,
+  Card,
+  CardTitle,
+  CardBody,
+  Collapse,
+  FormGroup,
+  Row,
+  Col
+} from 'reactstrap'
+
+function RequestForm (props) {
+  const jsonObject = props.item
+  const key = props.item._id
+
+  const Console3DLabel = [
```

Comment:
have you ever heard of an array of objects?

```js
const labelArray = [
  {lgSize: '3', title: 'Email/Contact', value: item.projectContact},
  // rest of your elements here
]
```

then in the map function do:
```jsx
{labelArray.map((item, itr) => {
   return (
       <Col md='12' lg={item.lgSize} key={itr}>
          <div className='label-2'>{item.title}</div>
           {item.value}
     </Col>
  )
})}
```

Reply:
If I am having a hard time trying to understand the code, everyone else will too. Please fix, or suggest an alternative